src/openrct2/interface/Colour.h

```diff
@@ -97,6 +97,7 @@ enum
     PALETTE_INDEX_162 = 162, //
     PALETTE_INDEX_171 = 171, // Saturated Red (lightest) Bright Red (middark)
     PALETTE_INDEX_172 = 172, // Saturated Red (10-11), Bright Red (midlight)
+    PALETTE_INDEX_173 = 173, //
```

Comment:
Add a comment to describe the palette, please.

Reply:
What would be a good comment? I'd like to match the previous comments but i'm unsure what this color would represent